docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -90,6 +90,68 @@ Once the initial enrollment has completed, and if Teleport Identity
 is enabled, the enrollment workflow will guide you through enabling SCIM
 provisioning as well.
 
+## Configure Access List Synchronization
+
+If Teleport Identity is enabled, the enrollment workflow will also guide you to configure syncing of
+User Groups and Direct Application Assignments as access lists. This will ensure
```

Comment:
Why is "Direct Application Assignments" a proper noun but "access lists" isn't?

Reply:
I've inverted this.